test/performance/odp_ipsecfwd.c

```diff
@@ -326,9 +326,10 @@ static void print_usage(void)
 	}
 
 	printf("\n"
-	       "Simple IPsec performance tester. Forward and process plain and ipsec packets.\n"
+	       "Simple IPsec performance tester. Forward and process plain and IPsec packets.\n"
+	       "\n"
+	       "Usage: %s [OPTIONS]\n"
 	       "\n"
-	       "Examples:\n"
```

Comment:
I think it would be good to keep the "Examples" title. The example command looks a bit weird without it. Maybe it would be also good to move the example & config file explanation section after the listing of the mandatory and optional options.

Reply:
Changed to use "E.g." in v2, as again this seems to be the style used in our testers. Also due to current style, better to keep the usage above the options sections.